src/demo/js/script.js

```diff
@@ -43,8 +43,10 @@ const preview = {
     if (params.type !== "json") {
       const repoLink = "https://git.io/streak-stats";
       const md = `[![GitHub Streak](${imageURL})](${repoLink})`;
+      const html = `<a href="${repoLink}"><img src="${imageURL}" alt="Readme Streak Stats" /></a>`;
```

Comment:
This works, but maybe we should use "GitHub Streak" as the alt to be consistent with the markdown version
```suggestion
      const html = `<a href="${repoLink}"><img src="${imageURL}" alt="GitHub Streak" /></a>`;
```